### ISSUE TITLE
Bad cropping of scores when posted to Discord

### ISSUE THREAD

**Issue by eddiephlash** [karl-run/among-us-stats#8]
Also, the individual character icons are both the same.

![image](https://user-images.githubusercontent.com/8182048/104400575-4af3cd80-5518-11eb-8430-c9557223db50.png)

**Comment by eddiephlash** [comment]
I think this only shows when there is an incomplete game

**Comment by karl-run** [comment]
Interesting. What browser are you using?

**Comment by eddiephlash** [comment]
Chrome. Windows.

**Comment by karl-run** [comment]
I managed to reproduce the icon issue, and it should be fixed by 42c6c379e322bef61d32a5ec4c5a603bacf913eb.

I'm not able to reproduce the weird cropping. What version of Chrome are you using? I'm on 87.0.4280.66 (Linux) and 87.0.4280.88 (Windows).

**Comment by eddiephlash** [comment]
Version 87.0.4280.141 (Official Build) (64-bit)

**Comment by karl-run** [comment]
![image](https://user-images.githubusercontent.com/1507032/104513124-73023000-55ef-11eb-8798-c21e8427a55b.png)

This is how it looks on 87.0.4280.141 when I try, even with an unfinished game. Can you see if you are able to reproduce the weird cropping now?

**Comment by eddiephlash** [comment]
The colors look great. 
I can no longer reproduce the weird cropping issue.

Thanks!

**Comment by karl-run** [comment]
I'm still seeing this cropping bug. 🤔

**Comment by eddiephlash** [comment]
Yes. I meant to check in here, but it slipped my mind. We were playing the other night and I noticed this between our 12th and 13th game. 

![12 games](https://cdn.discordapp.com/attachments/798921470125670422/821570926021050378/2021-03-17T022932.590Z.png)

![13 games](https://cdn.discordapp.com/attachments/798921470125670422/821576578281898004/2021-03-17T025200.221Z.png)

**Comment by karl-run** [comment]
Damn this is impossible to reproduce. I've tried two different OSes and a bunch of different combinations of stats.

If you ever come across a set of stats that consistently re-create this bug, if you don't mind, could you send me your local stored state?

You'll find it in the chrome dev tools here:

![image](https://user-images.githubusercontent.com/1507032/111842074-3ad4e180-88ff-11eb-8a17-519e86e00868.png)

Edit: You can mail them to be privately at [k@rl.run](mailto:k@rl.run)
